torch_geometric/distributed/local_graph_store.py

```diff
@@ -126,15 +184,18 @@ def from_partition(cls, root: str, pid: int) -> 'LocalGraphStore':
 
         if not meta['is_hetero']:
             attr = dict(edge_type=None, layout='coo', size=graph_data['size'])
-            graph_store.put_edge_index((graph_data['row'], graph_data['col']),
-                                       **attr)
+            graph_store.put_edge_index(
+                torch.stack((graph_data['row'], graph_data['col']), dim=0),
+                **attr)
```

Comment:
Why are we stacking here? put_edge_index expects a tuple. Same below.

Reply:
The reason :  edge_index (Tuple[torch.Tensor, torch.Tensor]) defined here can not pass through mp.spawn due to Tuple and used with stack to convert into tensor which can pass through mp.spawn.  We need send PartitionGraphStore/.. into subprocess.

Reply:
First time I hear that tuples are a problem with `mp.spawn`. Are you sure this info is correct?

Reply:
sure. i will check again and let my colleague check together. Also in from torch_geometric.testing import get_random_edge_index,  #L22 edge_index = torch.stack([row, col], dim=0) also used stack to convert into tensor instead of EdgeTensorTuple = Tuple[Tensor, Tensor]